### PR TITLE
Add security policy

### PR DESCRIPTION
**Pull request by RDIL**

### CODE REVIEW

File: docs/SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Security Policy
+
+## Reporting a Vulnerability
+
+If you find a security vulnerability in the Cirrus CI platform (the backend, web interface, etc.), please **follow the steps below**.
+
+1. **Do NOT comment about the vulnerability publicly.** *Doing so would be in violation of our Terms of Service, [and may have consequences](https://cirrus-ci.org/legal/terms/#cancellation-and-termination)!*
```
Comment (fkorotkov) [inline docs/SECURITY.md]: I think we can skip 

> *Doing so would be in violation of our Terms of Service, [and may have consequences](https://cirrus-ci.org/legal/terms/#cancellation-and-termination)!*

IMO looks scary 😅

Comment (RDIL) [reply docs/SECURITY.md]: Thats the point, we should show that public disclosure is a big no no

Comment (fkorotkov) [reply docs/SECURITY.md]: I think people who will read it will know it. People who disclose such issues don't read policies. 🤷‍♂

Comment (RDIL) [reply docs/SECURITY.md]: True

Comment (RDIL) [reply docs/SECURITY.md]: Fixed now

File: docs/SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Security Policy
+
+## Reporting a Vulnerability
+
+If you find a security vulnerability in the Cirrus CI platform (the backend, web interface, etc.), please **follow the steps below**.
+
+1. **Do NOT comment about the vulnerability publicly.**
+1. Please email [`hello@cirruslabs.org`](mailto:hello@cirruslabs.org) with the following format:
```
Comment (fkorotkov) [inline docs/SECURITY.md]:
```suggestion
2. Please email [`hello@cirruslabs.org`](mailto:hello@cirruslabs.org) with the following format:
```

File: docs/SECURITY.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  HOW TO EXPLOIT
+
+  Give exact details so our team can replicate it.
+
+  OTHER INFORMATION
+
+  If anything else needs to be said, put it here.
+  ```
+
+1. Please be patient. You will get an email back soon with further instructions.
```
Comment (fkorotkov) [inline docs/SECURITY.md]:
```suggestion
3. Please be patient. You will get an email back soon with further instructions.
```

File: docs/SECURITY.md (hunk 2 above)
Comment (RDIL) [reply docs/SECURITY.md]: Usually markdown would allow this but the code block may mess it up so I'll apply these

**Comment by RDIL** [comment]
Now markdownlint is mad

**Comment by RDIL** [comment]
how about we add this file to the mdlint ignore list

**Comment by fkorotkov** [comment]
See https://github.com/RDIL/cirrus-ci-docs/pull/1

**Comment by RDIL** [comment]
🦀 yes 🦀

**Pull request merged**